src/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="styles.css">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>A webpage about me</title>
+</head>
+<body>
+    <h1>Everything You Need to Know About Alex</h1>
```

Comment:
It's not always necessary for smaller web pages like this but it's a good idea to get into the habit of wrapping things in semantic tags like <header></header> <section></section> and/or <article></article>

Reply:
Awesome, thanks! I added these suggestions!